frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
```

Comment:
Should be imported through `/lib/primevue`

Reply:
exported RadioButton from frontend/src/lib/primevue/index.ts. Import works now. Thanks!